### PR TITLE
New slugify

### PR DESCRIPTION
**Pull request by vzuburlis**

### CODE REVIEW

File: src/blog/controllers/BlogCtrl.php [hunk 1]
```diff
@@ -7,7 +7,7 @@
 /**
 * The blog controller, get calls for display of posts
 */
-class Blog extends Controller
+class BlogCtrl extends Controller
```
Comment (panigrc) [inline src/blog/controllers/BlogCtrl.php:10]: I liked Blog class better 😒, BlogController would be nice too

File: src/core/classes/Sendmail.php [hunk 2]
```diff
@@ -9,7 +9,7 @@ function __construct($args)
     $args['subject'] = $args['subject']?? "Message from ".Gila::config('base');
     $args['headers'] = $args['headers']?? "From: GilaCMS <noreply@{$_SERVER['HTTP_HOST']}>";
     $args['message'] = $args['message']?? "";
-    if($args['message']=="") foreach(@$args['post'] as $key) {
+    if($args['message']==="") foreach(@$args['post'] as $key) {
```
Comment (panigrc) [inline src/core/classes/Sendmail.php:12]: you could also remove the error suppressing by doing `foreach($args['post'] ?? [] as $key)`

File: src/core/classes/View.php [hunk 3]
```diff
@@ -115,7 +115,7 @@ static function script($script, $uri=false, $prop='')
   {
     if(in_array($script, self::$script)) return;
     self::$script[]=$script;
-    if(Gila::config('use_cdn')=='1' && isset(self::$cdn_paths[$script])) {
+    if(@isset(self::$cdn_paths[$script])) {
```
Comment (panigrc) [inline src/core/classes/View.php:118]: error suppressing isn't needed so the line could be written `if(isset(self::$cdn_paths[$script])) {`

File: src/blog/controllers/BlogCtrl.php (hunk 1 above)
Comment (vzuburlis) [reply src/blog/controllers/BlogCtrl.php:10]: Doesnt matter, controller classes shall not be used from codebase from now on

**Pull request merged**